src/components/Forms/BluSearchBox.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+
+const BluSearchBox = () => {
+  return (
+    <div className="d-flex align-items-center">
+      <Form.Control
+        type="search"
+        placeholder="buscale prro"
```

Comment:
I know this is dummy text just yet but don't forget to use props instead of "buscale perro"